dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 --editable .
 
-Django == 1.8
-nose == 1.3.6
-coverage == 3.7.1
+Django == 1.11.5
```

Comment:
I guess we could take the same approach as before, i.e. `1.11` to get the latest revision in the `1.11` series